include/ignition/physics/GetEntities.hh

```diff
@@ -213,6 +213,56 @@ namespace ignition
       };
     };
 
+    /////////////////////////////////////////////////
+    /// \brief This feature retrieves the nested model pointer from the parent
+    /// model by specifying the model index and model index/name.
```

Comment:
I think this is probably supposed to say

```
/// \brief This feature retrieves the nested model pointer from the parent
/// model by specifying the index or name of the nested model.
```

From the user's point of view, the parent model does not need to be specified, so I would recommend not mentioning that in the public API documentation, as it may be misleading.

Reply:
Changed to say "...name or index of the nested model". https://github.com/ignitionrobotics/ign-physics/pull/228/commits/b836a0aa0fc7e9407e9f9d569491d3b9d32d9f82